pyproject.toml

```diff
@@ -8,13 +8,16 @@ authors = ["Łukasz Sójka", "Takuya ASADA"]
 arms = 'scylla_arms.main:program.run'
 
 [tool.poetry.dependencies]
-python = "^3.8"
```

Comment:
why fixing python and other versions in pyporoject.toml?
this may cause lots of problems when installing scylla-arms (it will be forcing these versions)

Reply:
Done

Reply:
we should keep it with ^ like it was before:
```[tool.poetry.dependencies]
python = "^3.8"
invoke = "^1.6.0"
pydantic = "^1.8.2"